File: abc/abc120/abc120_a.go
// https://atcoder.jp/contests/abc120/submissions/16643303

package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

const (
	initialBufSize = 1e4
	maxBufSize     = 1e8
)

var sc = bufio.NewScanner(os.Stdin)

func init() {
	buf := make([]byte, initialBufSize)
	sc.Buffer(buf, maxBufSize)
	sc.Split(bufio.ScanWords)
}

func next() string {
	sc.Scan()
	return sc.Text()
}

func nextInt() int {
	a, _ := strconv.Atoi(next())
	return a
}

func min(x, y int) int {
	if x > y {
		return y
	} else {
		return x
	}
}

func main() {
	A, B, C := nextInt(), nextInt(), nextInt()
	fmt.Println(min(C, B/A))
}

File: abc/abc120/abc120_a.cs
// https://atcoder.jp/contests/abc120/submissions/9316922

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var input = Console.ReadLine().Split().Select(int.Parse).ToArray();
            var A = input[0];
            var B = input[1];
            var C = input[2];

            Console.WriteLine(Math.Min(C, B / A));
        }
    }
}
